gtkm/auth/database/user_schema.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel
+
+
+class User(BaseModel):
```

Comment:
Mam jeszcze jedno pytanie, czy te modele (np użytkownika), które się zdefiniowało można wykorzystywać także przy wykonywaniu zapytań?

Reply:
@lapankrz   
Chodzi mi o to, czy inne node'y mogłyby w jakiś sposób korzystać z tej klasy